### ISSUE TITLE
Large gap between train-val vs test accuracies.

### ISSUE THREAD

**Issue by yash0307** [WenmuZhou/crnn.gluon#9]
Hello,

I trained the model on the 8M synthetic dataset by Max Jaderberg et al. for 20 epochs on 4 GPUs in parallel (total batch-size of 512). The training accuracy towards the end of the training was around 95% and the validation accuracy (on ICDAR'13 train set) was around 82%. However, when I test the model independently using 'predict.py' on the test set of ICDAR'13, I get around 53% accuracy. Do you know why this could be happening ? Is the batch-size somehow messing this up at the inference time ?

**Comment by WenmuZhou** [comment]
release code in https://github.com/WenmuZhou/crnn.gluon/blob/master/predict.py#L104-L106 and test

**Comment by WenmuZhou** [comment]
This accuracy is lower than crnn

**Comment by yash0307** [comment]
Tried that, the test accuracy raises to 59%, but still not close to train/val scores. Any other insights ?

**Comment by WenmuZhou** [comment]
use synth dataset pre-trained model

**Comment by ghost** [comment]
@yash0307 updates?
